docs/pages/server-access/guides/openssh.mdx

```diff
@@ -340,6 +337,96 @@ authenticate the host via the certificate we generated earlier.
 
 </Details>
 
+<Details title="Experiencing latency in leaf cluster Nodes?">
+
+### Proxy Jump
+
+In the generated OpenSSH client configuration, the `ProxyCommand` for each leaf 
+cluster connects through the root cluster's Proxy Service. In scenarios where
+leaf cluster Proxy Services are reachable by SSH client, you might prefer to
+connect directly through the leaf proxies to achieve lower latency.
+
+To achieve this, update the `ProxyCommand` of the leaf cluster's configuration
```

Comment:
```suggestion
To enable direct connections to a Proxy Service in a leaf cluster, open the 
SSH configuration file you generated earlier and update
the `ProxyCommand` of the leaf cluster's configuration
```

I would specify that this configuration block is in the SSH config the reader generated earlier in this guide. If a reader is following the guide, the configuration file they generated won't be in the default location.